app/serializers/course_serializer.rb

```diff
@@ -1,8 +1,28 @@
 class CourseSerializer < ActiveModel::Serializer
   # attributes to be serialized
-  attributes :id, :title, :short_title, :hyperlink, :semester_type, :semester_year, :allow_enrollment
-  # model association
+  attributes :id, :title, :short_title, :hyperlink, :semester_type, :semester_year, :allow_enrollment, :permissions
   has_many :sprints
```

Comment:
All of these need to be dependent on the read permission of the child resource (see link in ticket)